.circleci/config.yml

```diff
@@ -158,6 +158,7 @@ jobs:
                       - v0.6-repository_consistency
             - run: pip install --upgrade pip
             - run: pip install .[all,quality]
+            - run: pip install pytest
```

Comment:
Missed that in my first review. I don't think we need to add this here.

Reply:
(I could not remove it as the check imports every thing, it imports the HfDoctestParser, and this fails because you need pytest for that !

Reply:
OK